openpgp/src/main/java/com/yubico/yubikit/openpgp/Pw.java

```diff
@@ -18,8 +18,6 @@
 
 public enum Pw {
     USER((byte) 0x81), RESET((byte) 0x82), ADMIN((byte) 0x83);
-    public static final char[] DEFAULT_USER_PIN = "123456".toCharArray();
-    public static final char[] DEFAULT_ADMIN_PIN = "12345678".toCharArray();
```

Comment:
This is technically backwards incompatible. Also, what is the reason to remove these?

Reply:
the reason to move these to the places where the constants are used was to remove [MS_PKGPROTECT](https://spotbugs.readthedocs.io/en/latest/bugDescriptions.html#ms-field-should-be-package-protected-ms-pkgprotect) warning.

Reply:
This is a breaking change, I will revert this and dismiss the warning in GH.